vault/core.go

```diff
@@ -493,6 +493,14 @@ type Core struct {
 	secureRandomReader io.Reader
 
 	recoveryMode bool
+
+	clusterNetworkLayer cluster.NetworkLayer
+  
+	// PR1103disabled is used to test upgrade workflows: when set to true,
+	// the correct behaviour for namespaced cubbyholes is disabled, so we
+	// can test an upgrade to a version that includes the fixes from
+	// https://github.com/hashicorp/vault-enterprise/pull/1103
+	PR1103disabled bool
```

Comment:
Is this deliberately included in this PR?

Reply:
Yes, since it's in an OSS file i went ahead and ported it over.

Reply:
Actually i think we can move this to another location, i'll remove it from this PR